src/pages/NewMemberConsole.js

```diff
@@ -0,0 +1,79 @@
+import { useState, useEffect, useCallback } from 'react';
+import { div, h, img } from 'react-hyperscript-helpers';
+import { DAR } from '../libs/ajax';
+import SearchBar from '../components/SearchBar';
+import { Notifications, getElectionDate, processElectionStatus, darSearchHandler } from '../libs/utils';
+import { Styles} from '../libs/theme';
+import DarTable from '../components/dar_table/DarTable';
+import lockIcon from '../images/lock-icon.png';
+import { updateLists as updateListsInit } from '../libs/utils';
+import { tableHeaderTemplate, tableRowLoadingTemplate } from '../components/dar_table/DarTable';
+import DarTableSkeletonLoader from '../components/TableSkeletonLoader';
+
+export default function NewMemberConsole(props) {
+  const [electionList, setElectionList] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+  const [tableSize, setTableSize] = useState(10);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        setIsLoading(true);
+        const pendingList = await DAR.getDataAccessManageV2();
+        const openElectionList = (pendingList).filter((i) => {
+          return i.election.status === "Open";
+        });
+        setElectionList(openElectionList);
+        setFilteredList(openElectionList);
+        setIsLoading(false);
+      } catch (error) {
+        Notifications.showError({text: 'Error: Unable to retrieve data requests from server'});
+      }
+    };
+    init();
+  }, []);
+
+  const getUpdateLists = useCallback(() => {
+    return updateListsInit(filteredList, setFilteredList, electionList, setElectionList, currentPage, tableSize);
+  }, [filteredList, electionList, currentPage, tableSize]);
+
+
+  const handleSearchChange = darSearchHandler(electionList, setFilteredList, setCurrentPage);
+
+  return (
+    div({style: Styles.PAGE}, [
+      div({style: {display: "flex", justifyContent: "space-between"}}, [
+        div({className: "left-header-section", style: Styles.LEFT_HEADER_SECTION}, [
+          div({style: Styles.ICON_CONTAINER}, [
+            img({
+              id: 'lock-icon',
+              src: lockIcon,
+              style: Styles.HEADER_IMG
+            })
+          ]),
+          div({style: Styles.HEADER_CONTAINER}, [
+            div({style: Styles.TITLE}, ["DAC Member Console"]),
+            div({style: Styles.SMALL}, ["Should data access be granted to this applicant?"])
+          ]),
+        ]),
+        h(SearchBar, {handleSearchChange, currentPage})
+      ]),
+      h(DarTable, {
```

Comment:
There's nothing wrong here, just a UI improvement that I just thought of. When I made the table components I never made the Vote label adapt to whether or not the user had already submitted a vote. It never bothered me with the Chair console since their vote is essentially the final verdict, making the election ```Accepted``` or ```Closed```. This in turns changes the row's output and hides the vote button. However with the Member console the Vote button stays up, so it's hard for members to know whether or not they already voted on an election.

I don't think you'll have to do too much work, just some minor adjustments to ```DarTableActions``` and ```DarTableVoteButton```. The votes are already filtered out by user on line 98 in ```DarTableActions```, and the Vote Button is rendered out in that same function, so you should have everything you need to make this work.

The label update should be specific to the Member Console, since the Chairperson can't update their Chair vote once submitted. (Admin Console doesn't even render the Vote Button, so you have nothing to worry about there).

Reply:
@raejohanek @JVThomas 
> The label update should be specific to the Member Console, since the Chairperson can't update their Chair vote once submitted. (Admin Console doesn't even render the Vote Button, so you have nothing to worry about there).

A chair can vote as a member and then navigate back to their console, never having submitted a final vote. I think we could use the previously-voted treatment on both console pages.

Reply:
So do we want to change the vote button to a 'Voted' button? That could be confusing as a chair if they have voted as a member but not yet as a chair. We could do a 'voted' button on the member console and like a 'voted as member' button on the chair page?

Reply:
That will work, you'll just have to reference ```consoleType``` in ```DarTableActions``` to make sure the label is aded correctly. 

Also maybe change the phrase to "Edit Member Vote" or "Update Member Vote" (or anything along those lines) to show that that button performs an action.

Reply:
Good idea. So any member can update their vote? What if just across both the member and chair console pages if the member vote has been cast then the button says 'update vote'? Because I think on the chair console a button like 'update member vote' doesn't really capture the entire function of the button since it can also be used to vote as a chair.

Reply:
Yeah, good point. Let's go with "Update Vote".

Reply:
FYI, see https://broadworkbench.atlassian.net/browse/DUOS-1218 for more information on button text. That ticket contains some NIH feedback on labeling that's relevant to this discussion. It might be worth not addressing text label issues here and instead save that for DUOS-1218.